tests/test_modeling_wav2vec2.py

```diff
@@ -317,7 +319,17 @@ def test_initialization(self):
             model = model_class(config=configs_no_init)
             for name, param in model.named_parameters():
                 if param.requires_grad:
-                    if "conv.weight" in name or "masked_spec_embed" in name:
+                    if any(
+                        [
+                            x in name
+                            for x in [
+                                "conv.weight",
+                                "masked_spec_embed",
+                                "codevectors",
+                                "quantizer.weight_proj.weight",
+                            ]
+                        ]
+                    ):
```

Comment:
This is unreadable with the formatting. Make a list with the layer names in a separate variable, so we can put everything back on one line.